docs/resources/aws_s3_bucket_object.md

```diff
@@ -0,0 +1,81 @@
+---
+title: About the aws_s3_bucket_object Resource
+---
+
+# aws_s3_bucket_object
+
+Use the `aws_s3_bucket_object` InSpec audit resource to test properties of a single AWS bucket object.
+
+
+<br>
+
+## Limitations
+
+S3 object security is a complex matter.  For details on how AWS evaluates requests for access, please see [the AWS documentation](https://docs.aws.amazon.com/AmazonS3/latest/dev/how-s3-evaluates-access-control.html).  S3 buckets and the objects they contain support three different types of access control: bucket ACLs, bucket policies, and object ACLs.
+
+As of January 2018, this resource supports evaluating bucket object ACLs. In particular, users of the `be_public` matcher should carefully examine the conditions under which the matcher will detect an insecure bucket.  See the `be_public` section under the Matchers section below.
```

Comment:
I'd suggest not saying "bucket object ACLs" - say merely "object ACLs" for clarity.